sdk/purview/purview-account-rest/samples-dev/collections.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how get a list of typedefs
```

Comment:
Looks like we need to update this description and the summary below

Reply:
updated.